packages/newsroom-manager/src/RosterMember.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from "react";
+import { connect, DispatchProp } from "react-redux";
+import { colors, Checkbox, ToolTip, TextInput, TextareaInput, buttonSizes } from "@joincivil/components";
+import { EthAddress, RosterMember as RosterMemberInterface } from "@joincivil/core";
+import styled from "styled-components";
+import { FormSubhead, FormRow, FormRowItem, HelperText, TertiaryButton } from "./styledComponents";
+import { StateWithNewsroom } from "./reducers";
+import { UserData } from "./types";
+
+export interface RosterMemberProps {
+  newsroomAddress: EthAddress;
+  user: UserData;
+  onRoster: boolean;
+  onContract?: boolean;
+  newUser?: boolean;
+  updateRosterMember(onRoster: boolean, member: Partial<RosterMemberInterface>, newUser?: boolean): void;
+}
+
+export interface RosterMemberState {
+  newUserAddress?: EthAddress;
+}
+
+const Wrapper = styled.div`
+  padding: 20px 0;
+  border-top: 1px solid ${colors.accent.CIVIL_GRAY_4};
+`;
+
+const DisplayName = FormSubhead.extend`
```

Comment:
no extend https://github.com/styled-components/styled-components/issues/1546

Reply:
regex'd all of them out from whole repo